browser/extensions/api/brave_rewards_api.cc

```diff
@@ -606,6 +606,8 @@ ExtensionFunction::ResponseAction BraveRewardsSaveAdsSettingFunction::Run() {
   if (ads_service_) {
     if (params->key == "adsEnabled") {
       ads_service_->SetEnabled(params->value == "true");
+    } else if (params->key == "shouldOptOutOfAdConversions") {
```

Comment:
`"shouldOptOutOfAdConversions"` I'm guessing this is new, so can this become a const string declared somewhere?

Reply:
Fixed